reporting_dashboard.tf

```diff
@@ -0,0 +1,73 @@
+locals {
+  reporting_helper_lambda_name = "${local.environment}-dr2-reporting-helper"
```

Comment:
This name seems a bit vague; I called mine "ingest timings". Maybe we can call it something like "dr2-ingest-timings-report-lambda"?

Reply:
I thought this would be multiple metrics that we want to surface to dashboard. Ingest timing is just the first one. Maybe I can simply change it to "dr2-ingest-reporting"